MVP/defaults-build.yaml

```diff
@@ -26,6 +26,8 @@
         - timestamps
         - build-name:
             name: ${{ENV,var="name"}}-${{ENV,var="version"}}-${{ENV,var="release"}}
+        - inject:
+            script-content: echo "create_time=$(date --utc +%FT%TZ)" >> $WORKSPACE/job.properties
```

Comment:
Yes, you are right, it would be overwritten.  I didn't notice that line in sample_job.yaml, good catch.  So, what do you think?  Should I leave this here and change the ">" to ">>" in sample_job.yaml?  That would only be useful for people who have no jenkins jobs yet and are just beginning to create one with sample_job.yaml as their starting point.